app/generator/content/schedule/part2/index.js

```diff
@@ -1,8 +1,18 @@
 const getReductionDescription = require('./get-reduction-description')
 const getTopUpDescription = require('./get-top-up-description')
+const getReductionZeroDescription = require('./get-reduction-zero-description')
+const getRecoveryDescription = require('./get-recovery-description')
 
 const part2 = (schedule) => {
-  return schedule.adjustment.adjustmentValue > 0 ? getTopUpDescription(schedule) : getReductionDescription(schedule)
+  const remainingAmount = schedule.remainingAmount
+  switch (true) {
+    case (remainingAmount > 0):
+      return schedule.adjustment.adjustmentValue > 0 ? getTopUpDescription(schedule) : getReductionDescription(schedule)
+    case (remainingAmount === 0):
+      return getReductionZeroDescription(schedule)
+    case (remainingAmount < 0):
+      return getRecoveryDescription(schedule)

```

Comment:
Do we have existing functionality for recovery?

Why do we need to check and implement recovery functionality for this ticket?

Can we just leave this PR to handle the 1st (as-is) and 2nd (ticket ask) case functionality and leave this recovery aspect for the recovery ticket?

Like, we're adding dummy functionality for recovery when we have a sep ticket to target and focus on it with the signed off version.

From a testing POV because this exists, we need to test it which means QA needs to test this but it doesn't fit under the ticket's ask and expands its scope.

Reply:
Removed all instances of recovery.